vsts/pipelines/templates/_signBinary.yml

```diff
@@ -10,7 +10,7 @@ steps:
   displayName: 'Set sign binary variable'
   condition: >
    and(in(variables['Build.Reason'], 'Schedule', 'Manual'), eq(variables['Build.DefinitionName'], 'Oryx-CI'), 
-   or(startsWith(variables['Build.SourceBranch'], 'refs/heads/master'), startsWith(variables['Build.SourceBranch'],'refs/heads/patch/' )))
+   or(startsWith(variables['Build.SourceBranch'], 'refs/heads/master'), startsWith(variables['Build.SourceBranch'],'refs/heads/kichalla/' )))
```

Comment:
Maybe we should keep it as `patch` and avoid using personal branch name?

Reply:
That was a temporary change to make my branch be built using CI build definition.